# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h
# Commit: 0e3bc029e59f
# Full Hash: 0e3bc029e59f5f0319a57c469d7c0991081379ff
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-10 21:43:01
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 8b2ffdf05455 -r 0e3bc029e59f toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h
--- a/toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h	Tue Jun 10 11:49:57 2025 +0000
+++ b/toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h	Tue Jun 10 11:49:57 2025 +0000
@@ -25,7 +25,6 @@
   static StaticRefPtr<IdentityCredentialRequestManager> sSingleton;
   IdentityCredentialRequestManager() {};
   ~IdentityCredentialRequestManager() = default;
-
 };
 
 }  // namespace mozilla